### ISSUE TITLE
Various crate docs aren't building as of v1.10.7

### ISSUE THREAD

**Issue by CriesofCarrots** [solana-labs/solana#24361]
#### Problem
Docs for crates that depend on solana-ledger (I think) aren't building in v1.10.7+
eg:https://docs.rs/crate/solana-validator/1.10.7/builds/539266

```
[INFO] [stderr] error: expected None-delimited group
[INFO] [stderr]   --> /opt/rustwide/cargo-home/registry/src/github.com-1ecc6299db9ec823/solana-ledger-1.10.7/src/builtins.rs:32:25
[INFO] [stderr]    |
[INFO] [stderr] 32 |             to_builtin!(solana_bpf_loader_upgradeable_program!())
[INFO] [stderr]    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[INFO] [stderr]    |
[INFO] [stderr]    = note: this error originates in the macro `solana_bpf_loader_upgradeable_program` (in Nightly builds, run with -Z macro-backtrace for more info)
```

#### Proposed Solution
Triage and fix

**Comment by CriesofCarrots** [comment]
Actually, it looks like solana-ledger first failed at v1.10.6

**Comment by mvines** [comment]
I ran into this during https://github.com/solana-labs/solana/pull/24202

The nightly from April 8th exhibited this failure, April 1st nightly was happy though. I didn't debug further and assumed this was just a bug in nightly that'd be fixed upstream sometime before 1.62.0 hits stable

**Comment by CriesofCarrots** [comment]
Ah, thanks! Saved me some time. I guess I'll post an issue on rustlang repo; it's a shame that it's affecting all our docs builds.
